test/unit/overlay.js

```diff
@@ -6,19 +6,78 @@ var sharp = require('../../index');
 
 sharp.cache(0);
 
-// Main
+
+// Constants
+var MAX_ALLOWED_IMAGE_MAGICK_MEAN_SQUARED_ERROR = 0.3;
```

Comment:
Let me know if this fails again on other platforms or versions. We can adjust it upwards then.